bot/cogs/rules.py

```diff
@@ -0,0 +1,139 @@
+import re
+from typing import Optional
+
+from bot.constants import Channels, Roles
+from bot.decorators import redirect_output
+from bot.pagination import LinePaginator
+
+from discord import Colour, Embed
+from discord.ext.commands import Bot, Context, command
```

Comment:
The order and structure of imports are meant to be:
```py
import stdlib
from stdlib import lib

import thirdparty
from thirdparty import lib

import local
from local importlib
```

You have the `bot.` local group before the thirdparty group containing `discord.` and will need to be switched around.

Reply:
Let's thank Github Mobile for allowing me to switch to Desktop mode in order to change that now before I forget about it later.